app/helpers/license_helper.rb

```diff
@@ -3,15 +3,20 @@
 
 module LicenseHelper
   def license_select(name, selected = nil, opts = {})
-    select_tag(name, options_for_select(license_options(opts), selected), opts)
-  end
-
-  def grouped_license_select(name, selected = nil, opts = {})
-    select_tag(name, grouped_options_for_select(grouped_license_options(opts), selected), opts)
+    opts[:data] ||= {}
+    opts[:data]['role'] ||= 'seek-license-select'
+    recommended = opts.delete(:recommended)
+    source = opts.delete(:source) || Seek::License.combined
+    if recommended
+      options = grouped_options_for_select(grouped_license_options(source.values, recommended), selected)
+    else
+      options = options_for_select(license_options(source.values), selected)
+    end
+    select_tag(name, options, opts)
```

Comment:
would be good if this could be done via the `objects_input` helper, in the interest of keeping things self contained and easier to understand and maintain in the future. I can see the grouped options would be a problem here, but may be something useful elsewhere in the future. Did you give this any thought?

I'm a bit uncomfortable having different `select2 `implementations in various places, it was a pain and error prone when switching over from `typeahead`

Reply:
Could use `objects_input`, but I think it needs some tweaks to make it more appropriate for selecting single values.

Reply:
It's currently just a regular select tag (not a multiple select combobox) so it only allows 1 selection by default.

Limiting to 1 is a possibility, but then you have to clear the value with the X icon before you can choose a different license, instead of just directly selecting another option, which is a bit awkward